docs_src/security/Ch-APIGateway.md

```diff
@@ -158,15 +175,14 @@ the Core Data microservice with curl command like this:
 
     curl http://<core-data-microservice-ip>:59880/api/v2/ping
 
-Once the API gateway is started and initialized successfully, and all
-the common ports for EdgeX microservices are blocked by disabling the
-exposed external ports of the EdgeX microservices through updating the
-docker compose file, the EdgeX microservice will be behind the gateway.
-At this time both the microservice host/IP Address
-(\<core-data-microservice-ip\> in the example) as well as the service
-port (59880 in the example) are not available to external access. EdgeX
-uses the gateway as a single entry point for all the REST APIs. With the
-API gateway in place, the curl command to ping the endpoint of the same
+Where (\<core-data-microservice-ip\> is the Docker IP address of
```

Comment:
Not introduced by this PR, but I don't know why `<>` are being escaped.
```suggestion
Where (`<core-data-microservice-ip>` is the Docker IP address of
```

Reply:
I tend to leave markdown syntax I don't understand alone.... but changed anyway.

Reply:
Fixed in https://github.com/bnevis-i/edgex-docs/commit/2fff06e6ba4522a0ffa6a91f3b1dc29a138b3c95